e2e/staging/anon-donation-custom.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { test, expect } from '@playwright/test'
```

Comment:
it would be nice if local and staging are actually exactly the same tests with the only difference being the base URL.

Reply:
I've tested against staging until we setup the local environment. Still not convinced we actually need to test against staging in real conditions, but if we do the tests should be the same with only difference the URL as you suggested.